eng/common/scripts/Helpers/Package-Helpers.ps1

```diff
@@ -90,4 +90,90 @@ function CompatibleConvertFrom-Yaml {
   else {
       return ConvertFrom-Yaml $content
   }
+}
+
+<#
+.SYNOPSIS
+Common function that will verify that the YmlFile being loaded exists, load the raw file and
+return the results of CompatibleConvertFrom-Yaml or report an exception and return null if
+there's a problem loading the yml file. The return is the PowerShell HashTable object.
+
+.DESCRIPTION
+Common function that will verify that the YmlFile being loaded exists, load the raw file and
+return the results of CompatibleConvertFrom-Yaml or report an exception and return null if
+there's a problem loading the yml file. This is just to save anyone needing to load yml from
+having to deal with checking the file's existence and ensure that the CompatibleConvertFrom-Yaml
+is made within a try/catch. The return is the PowerShell HashTable object from the
+CompatibleConvertFrom-Yaml call or $null if there was an issue with the convert.
+
+.PARAMETER YmlFile
+The full path of the yml file to load.
+
+.EXAMPLE
+CompatibleLoadAndConvertFrom-Yaml -YmlFile path/to/file.yml
+#>
+function LoadAndCompatibleConvertFrom-Yaml {
```

Comment:
The naming here is a bit weird. The name in the `.Example` is right, but the name of the actual function is wrong. `LoadAndCompatibleConvertFrom-Yaml`. I think the name in the example makes more sense.

OR, you totally rename it to `LoadFrom-Yaml` or something like that.

Reply:
I'm just going to rename it LoadFrom-Yaml as you suggest :) It's simpler.

Reply:
Renamed with 3rd commit.